TFNetworkRecLayer.py

```diff
@@ -4177,3 +4177,141 @@ def call(self, *args, **kwargs):
     return y
 
 
+class LayerNormLSTMCell(BaseRNNCell):
+  """LSTM unit with layer normalization and recurrent dropout
+
+  Layer normalization implementation is based on:
+        https://arxiv.org/abs/1607.06450
```

Comment:
Actually, this is only one of the variants you cover, right?
You should explain that in the description.
E.g. say sth like: With `global_norm=True`, `per_gate_norm=...`, ... (etc) (and other params left as default) you get the original layer norm LSTM as presented here (link).